src/pybricks/_common.pyi

```diff
@@ -81,6 +81,7 @@ class Motor(DCMotor):
     def track_target(self, target_angle: int) -> None: ...
 
 class Speaker:
+    def volume(self, volume: Optional[int]=None) -> Optional[int]: ...
```

Comment:
We've been using `@overload` for these sorts of methods since the return type is a function of the parameters.
```suggestion
    @overload
    def volume(self) -> int: ...
    @overload
    def volume(self, volume: int) -> None: ...
    def volume(self, *args) ...
```

Reply:
Done.